src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -97,11 +202,11 @@ const BudgetCategoriesTableContainer = (props) => {
                     filter: {
                         def_codes: defCodes.map((defc) => defc.code)
                     },
-                    spending_type: apiSpendingTypes[spendingCategory],
+                    spending_type: 'total',
```

Comment:
This should still dynamically refer to either total spending or award spending based on the state of the dropdown